openapi.yaml

```diff
@@ -1914,8 +1914,14 @@ paths:
         schema:
           type: array
           items:
+            enum:
+            - 'true'
+            - 'false'
             type: string
-            minLength: 1
+            description: |-
+              * `true` - true
+              * `false` - false
+        description: true if the learning resource offers a certifacate
```

Comment:
This OpenAPI spec is odd. I would expect the boolean parameter to have `type: boolean`.

In `LearningResourcesSearchRequestSerializer`, what do you think of either:

1.  using `certification = serializers.BooleanField(required=False)`
2. OR `certification = StringArrayField(child_serializer=serializers.BooleanField(), ...)`.

(1) seems better to me (no need to pass an array of booleans; no filter applied is the same as `certification=true,false`). But I see that `generate_filter_clauses` expects an iterable, and if we need to keep it that way, (2) seems OK, too.